README.md

```diff
@@ -0,0 +1,47 @@
+# Epic Board Game Shop
+
+## Opis Projektu
+
+Epic Board Game Shop to aplikacja umożliwiająca zarządzanie zamówieniami oraz produktami w sklepie stacjonarnym z 
+grami planszowymi. Klient przychodzi do sklepu, gdzie wraz ze sprzedawcą tworzą zamówienie, 
+które następnie jest przetwarzane przez aplikację. 
+Program integruje się z zewnętrznym systemem magazynowym [Legendary Warehouse](https://github.com/WojciechBarwinski/legendary-warehouse), 
```

Comment:
Program integruje się z zewnętrznym systemem [Legendary Warehouse](https://github.com/WojciechBarwinski/legendary-warehouse), 
który symuluje pracę magazynu.

Tak bym to napisał